sema/discovery/__main__.py

```
@@ -132,16 +132,35 @@ def normalise_mime_type_requests(request_mimes: Iterable | str) -> str:
+    request_mimes = request_mimes.split(",") if request_mimes else []  # type: ignore # noqa
+    return ",".join({normalise_mime_type_requests(mt) for mt in request_mimes})  # type: ignore # noqa
 
 
 def make_service(args: Namespace) -> Discovery:
     """Make the service with the passed args"""
+
+    url = args.url or args.url_option
+    if url is None:
+        raise ValueError(
+            "Discovery requires a URL."
+            "Provide it as a positional argument or use --url option"
```

Ruff

Trailing comma missing

Add trailing comma

(COM812)